webots/worlds/LineFollowerTrack.wbt

```diff
@@ -1,44 +1,46 @@
-#VRML_SIM R2023b utf8
-
-EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/backgrounds/protos/TexturedBackground.proto"
-EXTERNPROTO "https://raw.githubusercontent.com/cyberbotics/webots/R2023a/projects/objects/backgrounds/protos/TexturedBackgroundLight.proto"
-EXTERNPROTO "../protos/Zumo32U4.proto"
-EXTERNPROTO "../protos/LineFollowerTrack.proto"
-IMPORTABLE EXTERNPROTO "../protos/Supervisor.proto"
-
-WorldInfo {
-  info [
-    "Example uses of Track nodes, as caterpillar tracks on a robot, or as a conveyor belt."
-  ]
-  title "Track"
-  basicTimeStep 8
-  contactProperties [
-    ContactProperties {
-      material1 "track material"
-      coulombFriction [
-        500
-      ]
-      softCFM 0.0001
-    }
-  ]
-}
-Viewpoint {
-  orientation 0.3787583266277594 0.09714544028170935 -0.9203830145339561 2.677946076660944
-  position 1.2569587324159737 0.7945022889765716 1.5111964909293643
-}
-TexturedBackground {
-}
-TexturedBackgroundLight {
-}
-LineFollowerTrack {
-}
-DEF ROBOT Zumo32U4 {
-  translation -0.24713614078815466 -0.04863962992854465 0.013994298332013683
-  rotation -1.0564747468923541e-06 8.746699709178704e-07 0.9999999999990595 1.5880805820884731
-  name "Zumo"
-}
-Supervisor {
-  name "Supervisor"
-  controller "Supervisor"
-  supervisor TRUE
-}
+#VRML_SIM R2023b utf8
```

Comment:
Line endings? It should be LF as it's the standard of Webots, but I am not sure what it is right now.

Reply:
I saved the file with webots, so now its LF. It was CRLF before.